source/clear-linux/concepts/autospec-about.rst

```diff
@@ -1,112 +1,106 @@
-.. _autospec-about: 
+.. _autospec-about:
 
 Autospec
 ########
 
-.. _incl-autospec-overview:
+A standard RPM build process using ``rpmbuild`` requires a tarball and .spec
+file to start. The .spec provides information about a package, how to
+build it, as well as what should be installed and where.
 
-Overview
-********
+``autospec`` is a tool to assist in the automated creation and maintenance of
+RPM packaging in |CL-ATTR|. Compared to ``rpmbuild``, ``autospec`` requires
+only a tarball and package name to start.
 
-Whereas a standard RPM build process using ``rpmbuild`` requires a tarball 
-and ``spec`` file to start, ``autospec`` only requires a tarball and package
-name. ``autospec`` analyzes the source code and :file:`Makefile` information 
-in order to generate a ``spec`` file for you. Although not required, you can 
-influence ``autospec`` by providing control files. 
+How autospec works
+******************
+
+``autospec`` attempts to infer the requirements of the .spec file
+by analyzing the source code and :file:`Makefile` information.
+It will continuously run updated builds based on new information
+discovered from build failures until it has a complete and valid .spec file.
+You may optionally influence the behavior of ``autospec`` by providing
```

Comment:
Please make this a note and remove "optionally." Something like: 
.. note:: 
   You may influence the behavior of ``autospec`` by providing :ref: `control files [...]  See Table 1.